<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngineConfigurationDialog</class>
 <widget class="QDialog" name="EngineConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="m_tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Name:</string>
         </property>
         <property name="buddy">
          <cstring>m_nameEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="m_nameEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Co&amp;mmand:</string>
         </property>
         <property name="buddy">
          <cstring>m_commandEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="m_commandEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_browseCmdBtn">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Working &amp;Directory:</string>
         </property>
         <property name="buddy">
          <cstring>m_workingDirEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="m_workingDirEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_browseWorkingDirBtn">
           <property name="toolTip">
            <string>Browse working directory.</string>
           </property>
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Protocol:</string>
         </property>
         <property name="buddy">
          <cstring>m_protocolCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="m_protocolCombo"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&amp;Init Strings:</string>
         </property>
         <property name="buddy">
          <cstring>m_initStringEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPlainTextEdit" name="m_initStringEdit">
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="m_whitePovCheck">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>This option should be checked for Xboard/Winboard engines that report their scores from white's perspective.</string>
         </property>
         <property name="text">
          <string>Scores from &amp;white's perspective</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="m_debugCheck">
         <property name="text">
          <string>Debug</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeView" name="m_optionsView">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="m_restoreBtn">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Restore the default value for each option</string>
           </property>
           <property name="text">
            <string>Restore defaults</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_detectBtn">
           <property name="toolTip">
            <string>Detect the engine's options and supported features</string>
           </property>
           <property name="text">
            <string>Detect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="m_progressBar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EngineConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
